out/blocks/wd_admin_order_list_colgroup.tpl

```diff
@@ -0,0 +1,4 @@
+<col width="10%">
+<col width="10%">
```

Comment:
Just curious if the UI doesn't break, as I suppose the sum of cols percentages now exceeds 100%.

Reply:
The total of percentages can exceed 100% in tables, so the UI won't break. I agree that it's a bit odd, but that's how tables work. ;)